tools/static-assets/server/esm-loader.mjs

```
@@ -0,0 +1,264 @@
+    }
+    dirs.push(path.join(serverDir, 'node_modules'));
+    packageNodeModulesMap[item.path] = dirs;
+  }
+
+  let currentPackagePath = null;
+
+  function resolveInDirs(name, dirs) {
+    for (const dir of dirs) {
+      try {
+        return createRequire(path.join(dir, '_v.js')).resolve(name);
+      } catch (e) { continue; }
```

ESLint

[error] Replace `·continue;` with `⏎········continue;⏎·····`

(prettier/prettier)